<!DOCTYPE html>
<html>
<head>
    <link rel="stylesheet" href="index.css">
    <meta name="google-site-verification" content="xvD194KsBxeaW40tr48z5nGiEF-zbgEgpTNHhK2AMu8" />
  </head>
<html>
  <style>
    body{
        margin: 5% 20%;
        font-family: "Palatino", serif;
        text-align: left;
    }

    hr {
      margin: 2% 0%;
    }

    li {
      margin-bottom: 1%;
      margin-top: 1%;
    }

    h1, h2 {
      text-align: center;
    }

    #picture-div {
      width: 350px; 
      float:right;
      margin-left: 10%;
      margin-right: -13%;
    }

    img{
      height: 250px;
      border-radius: 25px;
    }

    a {
      color: darkblue;
    }

  </style>
<body>

  <div id="picture-div"><img src="IMG_github.jpg"></img></div>

  <h1>Peter Halmos</h1>
<p> 

Hi! I’m a CS PhD student at Princeton University and am fortunate to be advised by <a href = “https://raphael-group.github.io/”> Ben Raphael </a>. I previously graduated from Columbia University as a Computer Science major with a concentration in Chemistry. At Columbia and the New York Genome Center I was fortunate to have met and worked with <a href = "https://davidaknowles.github.io/">David Knowles</a> who introduced me to the intersection of machine learning and computational biology.

<br> Contact: ph3641 [at] princeton [dot] edu | Office: 35 Olden Street #318b
</p>
<hr>

    <h2>Research Interests</h2>
    <ul>
My research lies at the intersection of pure machine learning and computational biology. I am interested in the computational and mathematical aspects of optimal transport, including low-rank and neural optimal transport, as well as flow-based generative models. My focus is on developing and adapting these tools to quantitatively infer developmental trajectories in single-cell and spatial transcriptomics, and to understand the dynamics of cell differentiation in space and time.
    </ul>
<hr>

<h2>Publications</h2>
    <ul>
<li> Learning Latent Trajectories in Developmental Time Series with
Hidden-Markov Optimal Transport </a> </li>
Peter Halmos, Julian Gold, Xinhao Liu, and Benjamin J. Raphael (in submission)
<li> <a href = "https://openreview.net/pdf?id=hGgkdFF2hR" > Low-Rank Optimal Transport through Factor Relaxation with Latent Coupling </a> </li>
Peter Halmos, Xinhao Liu, Julian Gold, and Benjamin J. Raphael (Neurips 2024)
<li> <a href = "https://www.biorxiv.org/content/10.1101/2024.03.05.583575v1" > DeST-OT: Alignment of Spatiotemporal Transcriptomics Data </a> </li>
Peter Halmos, Xinhao Liu, Julian Gold, Feng Chen, Li Ding, and Benjamin J. Raphael (Cell Systems, RECOMB 2024)
<li><a href = "https://www.ashg.org/wp-content/uploads/2022/09/ASHG2022-PosterAbstracts.pdf" > Pooled RNA-IP approach to investigate variant effects on RBP binding and splicing </a> </li>
Megan Schertzer, Peter Halmos, K. Dobrindt, K. Brennand, David Knowles (ASHG 2022)
<li><a href = "https://arxiv.org/abs/2308.11933" > System Identification for Continuous-time Linear Dynamical Systems </a> </li>
Peter Halmos, Jonathan Pillow, David Knowles (ArXiV)
    </ul>
<hr>
        

<h2>Previous Research Projects</h2>
<ul>
   <li>I also worked at <a href="https://www.foundationmedicine.com">Foundation Medicine</a> as an undergraduate, supervised by Justin Newberg, Garrett Frampton, and Megan Montesion. I worked on two projects accepted to AACR 2020 as abstracts
   <ul>
     <li><i>Pan-chromosome analysis does not reveal parental or ancestral bias in chromosome loss of heterozygosity</i> and <i>Pan-cancer analysis of sex differences and their associations with ancestry and genomic biomarkers in a large comprehensive genomic profiling dataset </i>
   </ul>
  </li>
    
</ul>
<hr>

<h2>Teaching</h2>
<p>
I was previously a teaching assistant for introductory Organic Chemistry at Columbia with Professor Talha Siddiqui. During the summers of 2023, I was a Computer Science instructor at the King Summer Institute, where I taught an introductory Java course. I taught 3 hours/weekday for the duration of the course, wrote daily problem sets, held instructor office hours, and meet with students one on one to address questions they had about the material. 

<!
Also, I have been a teaching assistant for the following courses at Princeton and Columbia University. My responsibilities included grading homework and exams, holding office hours and recitations, writing practice problems, and editing lecture notes.
</p>
<ul>
<li> COS 423 Theory of Algorithms, taught by Professor Robert Tarjan in Spring 2024</li>
<li> COS 521 Advanced Algorithm Design, taught by Professor Huacheng Yu in Fall 2023</li>
  <li>COMS 4995 Advanced Algorithms, taught by Professor Alexandr Andoni in Spring 2022</li>
  <li>CSOR W4231 Analysis of Algorithms, taught by Professor Alexandr Andoni in Fall 2021</li>
  <li>MATH UN2030 Ordinary Differential Equations, taught by Professor Evgeni Dimitrov in Fall 2021</li>
  <li>CSOR W4231 Analysis of Algorithms, taught by Professor Xi Chen in Summer 2021</li>
  <li>COMS W3261 Computer Science Theory, taught by Timothy Randolph in Summer 2021</li>
  <li>MATH GU4042 Modern Algebra II, taught by Professor Inbar Klang in Spring 2021</li>
  <li>COMS W3203 Discrete Math: Combinatorics and Graph Theory, taught by Professor Ansaf Salleb-Aouissi in Fall 2020 and Spring 2021</li>
</ul>
Lastly, I gave lectures and served as a TA for the Columbia Undergraduate Math Society's Introduction to Proofs workshop in 
<a href="http://www.math.columbia.edu/~ums/introproofs2020.html">Fall 2020</a> and <a href="http://www.math.columbia.edu/~ums/IntroToProofsFall2021.html">Fall 2021</a>.
This was a 4-week workshop series for undergraduate students that teaches how to write mathematical proofs.
<hr>
>

<h2>Other</h2>
Outside of my academic interests I am an avid hiker and an amateur birder, botanist, and <a href="https://www.flickr.com/photos/143173976@N05/">photographer</a>. Credits for this website template go to <a href="https://eg314159.github.io">Elena Gribelyuk</a>, who I shamelessly lifted this from.

    <hr>

</body>
</html>
